### ISSUE TITLE
Autogrill dies not work properly 

### ISSUE THREAD

**Issue by kolchose1980** [PhilippC/keepass2android#2132]
I use k2a long time on my Samsung S10 and now on my pixel7. 
But the auto fill works not properly. For example Google Play Store. Today I bought a App and I need the password but k2a not regonized that I have to put password in. 
Another KeePass App works.
I like k2a but if I have very often to open k2a manually and copy the password to clipboard is not user friendly.

This issue I have with Android 12 @ my Samsung S10 and also with Android 13 @ my pixel 7

Any ideas?

**Comment by Githerazi** [comment]
Have you tried the auto fill plugin?
https://github.com/PhilippC/keepass2android/blob/master/docs/Available-Plug-ins.md

**Comment by kolchose1980** [comment]
I tried to install but the plugin is not compatible

**Comment by grossgrundbesitzer** [comment]
Same problem here. Pixel 7 pro. Autofill does not work great. Most of the time only email address or username is pasted but not password. KeePass autofill Plugin cannot be installed.

**Comment by markpiazza** [comment]
Is this duplicate of this [issue](https://github.com/PhilippC/keepass2android/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc++autofill+#issue-1349719042) ?

Having the same problem on a Samsung S22+ with Android 13. Works just fine on my Pixel 3XL running Android 12. Autofill prompt doesn't pop-up in Gboard like it does on 12. It mostly gives me the same error, or it autofills the password in the username field. Works just fine in Forefox.

Basically, autofill seems to have broken between version of Android.

**Comment by litmich** [comment]
Same here,  since update to Android 13 (Note 10 lite) , with auto fill plugin.
With some apps works as intended.
Some only raise notification for auto fill,  but no popup, even after selecting "auto fill" submenu of the input field.
Some show suggestion inlined in a swift keyboard,  some not.
Some apps seem not to be recognised as target for autofill at all, no autofill prompt in any form. Only switch to key pass keyboard helps.

**Comment by dtar2** [comment]
I'm having the same issue. KP2A version 1.09d-r0 on Android 13.

Typically it will only autofill the username field, but not the password field.

**Comment by rojaxthegreat** [comment]
Same issue, pixel 7 android 13, KP2A v. 1.09e-r0

**Comment by LtJaeger** [comment]
Same thing here, with the SAMSUNG S23. Autofill works (kind of) for the username but most of the time doesn't for PW

**Comment by dtar2** [comment]
The easiest workaround is to switch to the Keepass2android keyboard when autofill doesn't work.

**Comment by markpiazza** [comment]
This isn't an easy workaround. It adds about 10 taps to any interaction
with password fields.

**Comment by PhilippC** [comment]
Closing as duplicate of #2059 . Please see https://github.com/PhilippC/keepass2android/issues/2059#issuecomment-1455921233 for how to report autofill issues if they persist to exist with 1.09e-r3
